docs/lib/datastore/fragments/ios/real-time/observe-snippet.md

```diff
@@ -22,6 +22,13 @@ func unsubscribeFromPosts() {
 }
 ```
 
+<amplify-callout>
+
+`DataStore.clear()` and `DataStore.stop()` will not remove any active subscriptions. The subscriber will be active and will continue to receive subscription events.
```

Comment:
why would the subscriber continue to receive subscription events if DataStore is stopped/cleared? It's not that the subscriber will get more events, it's that the subscription connection will not receive a completion signal. Perhaps we can clarify using language from https://developer.apple.com/documentation/combine/subscriber such as.. 


``` 
`DataStore.clear()` and `DataStore.stop()` will stop the DataStore sync engine and keep any subscriptions connected. There will not be any additional subscription events received by the subscriber until DataStore is started (DataStore.start) or the sync engine is re-initiated upon performing a DataStore operation (DataStore.query/save/publisher?)  
```
does DataStore restart if a subscriber is connected to a publisher?